### ISSUE TITLE
CUSTOMIZE IMAGE MKS TFT35

### ISSUE THREAD

**Issue by PaoloViper74** [makerbase-mks/MKS-DLC32#298]
Is it possible to change the start screen to a personalized one, as it was possible to do in the MKS TFT35?

**Comment by tbsmnea** [comment]
Yes, of course, if MKS (in quotation marks) would release the source code. But I think you can only wait for years. That is your business

**Comment by Mungbeanz** [comment]
They have released some firmware.  I see some display configs but didnt see the TFT35
[](url)https://github.com/makerbase-mks/MKS-DLC32-FIRMWARE

**Comment by PaoloViper74** [comment]
thank you, I've already edited the image, how do I reassemble in the fil bin?Inviato dal mio Galaxy
-------- Messaggio originale --------Da: Mungbeanz ***@***.***> Data: 15/11/22  04:08  (GMT+01:00) A: makerbase-mks/MKS-DLC32 ***@***.***> Cc: PaoloViper74 ***@***.***>, Author ***@***.***> Oggetto: Re: [makerbase-mks/MKS-DLC32] CUSTOMIZE IMAGE MKS TFT35 (Issue #298) 
They have released some firmware.  I see some display configs but didnt see the TFT35
https://github.com/makerbase-mks/MKS-DLC32-FIRMWARE

**Comment by Mungbeanz** [comment]
Sorry have not looked deeply into this. I noticed .ino files. Cant you compile with arduino ide and extract bin from your appdata folder or flash directly?

**Comment by PaoloViper74** [comment]
i managed to create the .bin file but besides being half the original size it doesn't work

**Comment by Medina-ui** [comment]
I have compiled the firmware, with no errors. TFT35, but the same happen. Half the size, uploaded the firmware to the board and besides LED blinking, no image on TFT, and no response after from the board. Good news, using an already compiled firmware board come back alive

**Comment by PaoloViper74** [comment]
For compiling you have to use VScode , but the source data contains many errors , old version.Br Paolo Inviato dal mio Galaxy
-------- Messaggio originale --------Da: Medina-ui ***@***.***> Data: 29/11/22  23:41  (GMT+01:00) A: makerbase-mks/MKS-DLC32 ***@***.***> Cc: PaoloViper74 ***@***.***>, Author ***@***.***> Oggetto: Re: [makerbase-mks/MKS-DLC32] CUSTOMIZE IMAGE MKS TFT35 (Issue #298) 
I have compiled the firmware, with no errors. TFT35, but the same happen. Half the size, uploaded the firmware to the board and besides LED blinking, no image on TFT, and no response after from the board. Good news, using an already compiled firmware board come back alive

**Comment by vondutchga** [comment]
try to upload it using platform.io directly (the little upload error next to the build button) make sure to set your comport in mksdlc32.ini it uploaded for me as long as I didnt touch it. I convert a jpg to a c file replaced mks_logo.c with the one I had and got an error. If somebody has the steps how to change the logo out I will see if I can figure out how to "burn" it into the board

**Comment by sharknetwork24** [comment]
There is a very simple way and that each of us can change very simply.
Soon I will insert a guide to replace the personalized start screen.
